crates/bevy_animation/src/lib.rs

```diff
@@ -115,14 +115,22 @@ impl Default for AnimationPlayer {
 
 impl AnimationPlayer {
     /// Start playing an animation, resetting state of the player
-    pub fn play(&mut self, handle: Handle<AnimationClip>) -> &mut Self {
+    pub fn start(&mut self, handle: Handle<AnimationClip>) -> &mut Self {
         *self = Self {
             animation_clip: handle,
             ..Default::default()
         };
         self
     }
 
+    /// Like [`start`](Self::start), but does nothing if the requested animation is already playing.
```

Comment:
this doc is not self-explaining, could you reformulate?

Reply:
Much better IMO.

Reply:
good for me, thanks 👍